app/jobs/daily_regeneration_job.rb

```diff
@@ -0,0 +1,19 @@
+class DailyRegenerationJob < ApplicationJob
+  queue_as :regeneration
+
+  def perform(school:)
+    total_amr_readings_before = AmrValidatedReading.count
```

Comment:
We don't really need this level of logging. We have this detail in the school job run by admins, to just give them some feedback about what has happened. But within the nightly job we typically just log that we've run each school, with no detail.

The `.count` can also be slow if the db stats are out of date.

Can you just simplify the logging to just record the start and end time for each job, e.g. puts "#{DateTime.now.utc} Regeneration for school #{school.name} start" and one for the end.